filesystem/node.go

```diff
@@ -43,25 +43,33 @@ var _ fs.HandleWriter = &Node{}       // Write
 var _ fs.HandleReader = &Node{}       // Read
 var _ fs.NodeFsyncer = &Node{}        // Fsync
 var _ fs.NodeRenamer = &Node{}        // Rename
+var _ fs.NodeSymlinker = &Node{}      // Symlink
+var _ fs.NodeReadlinker = &Node{}     // Readlink
 
 // Default permissions: we don't have any right now.
 const defaultPerms = 0755
 
+// All permissions.
+const allPerms = 0777
+
 // Maximum file size.
 const maxSize = math.MaxUint64
 
 // Node implements the Node interface.
+// ID, Mode, and Fastlink are currently immutable after node creation.
+// Size (for files only) is protected by mu.
 type Node struct {
 	cfs CFS
-	// These fields are immutable after node creation.
+	// ID is a unique ID allocated at node creation time.
 	ID uint64
 	// Used for type only, permissions are ignored.
 	Mode os.FileMode
+	// Fastlink is the path a symlink points to.
+	Fastlink string
```

Comment:
sure. but using `SymlinkTarget` instead, `Symlink` is already a `Symlinker` interface method.